### PR TITLE
[expo] bump bundled version of `react-native-svg`

### LINKED ISSUE #19013: React-native-svg typescript errors.
### Summary

`<G>` tags in from `react-native-svg` is throwing the typescript error `Type '{ children: Element[]; }' has no properties in common with type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<GProps, any, any>> & Readonly<GProps>'`. It seems to be related to https://github.com/react-native-svg/react-native-svg/issues/1856#issuecomment-1238617363 and is resolved in version `12.4.4`. Leveraging `patch-package` seems to be a temporary workaround for now.

### What platform(s) does this occur on?

Not Applicable

### Environment

 expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 12.5.1
      Shell: 5.8.1 - /bin/zsh
    Binaries:
      Node: 16.16.0 - /usr/local/bin/node
      Yarn: 1.22.19 - ~/.yarn/bin/yarn
      npm: 8.15.0 - /usr/local/bin/npm
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    IDEs:
      Android Studio: 2021.2 AI-212.5712.43.2112.8609683
      Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
    npmPackages:
      expo: ~46.0.9 => 46.0.10 
      react: 18.0.0 => 18.0.0 
      react-native: 0.69.5 => 0.69.5 
    npmGlobalPackages:
      eas-cli: 2.0.0
      expo-cli: 6.0.5
    Expo Workflow: managed

### Minimal reproducible example

https://github.com/iM-GeeKy/svg-typescript-error

### PR DESCRIPTION
**Pull request by Simek**
# Why

Fixes #19013

# How

Bump bundled `react-native-svg` to the latest release from `12` major via `et uvm`.

# Test Plan

NCL app was starting correctly after performing a bump.

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: ios/Exponent/Versioned/Core/Api/Components/Svg/Utils/RNSVGPathMeasure.m [hunk 1]
```diff
@@ -20,7 +20,7 @@
 /**
  * returns the distance between two points
  */
-CGFloat distance(CGPoint p1, CGPoint p2)
+static CGFloat distance(CGPoint p1, CGPoint p2)
```
Comment (trajano) [inline ios/Exponent/Versioned/Core/Api/Components/Svg/Utils/RNSVGPathMeasure.m:23]: Just curious, why is this change required?

Comment (trajano) [reply ios/Exponent/Versioned/Core/Api/Components/Svg/Utils/RNSVGPathMeasure.m:23]: And why is there no Android counterpart?

Comment (Simek) [reply ios/Exponent/Versioned/Core/Api/Components/Svg/Utils/RNSVGPathMeasure.m:23]: PR adding those changes is here:
* https://github.com/react-native-svg/react-native-svg/pull/1508
